package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-digitalbazaar",
-  "version": "0.0.1-0",
+  "version": "0.0.1",
```

Comment:
@aljones15 I think you need to rebase on master, some of the changes here like to the readme are already on master.  Also, the publish script takes care of bumping versions, so there is no need to modify the package.json in dev branches.

Reply:
yeah I thought that Pr comment was weird too with changing the version.

Reply:
btw master currently 0.0.1-0 which I believe will cause issues when bumping versions.

Reply:
@aljones15 maybe you need to pull master?  https://github.com/digitalbazaar/eslint-config-digitalbazaar/blob/master/package.json#L3